### PR TITLE
feat: optimism bedrock upgrade

### PR DESCRIPTION
**Pull request by kaladinlight**
TODO: add readiness probe for `op-node` after upgrade

### CODE REVIEW

File: pulumi/src/config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as pulumi from '@pulumi/pulumi'
 import { Config as BaseConfig, Dockerhub } from '.'
 
-const SUPPORTED_NETWORKS = ['mainnet']
+const SUPPORTED_NETWORKS = ['mainnet', 'goerli']
```
Comment (kaladinlight) [inline pulumi/src/config.ts:4]: remove

File: .circleci/config.yml [hunk 2]
```diff
@@ -534,17 +534,17 @@ aliases:
     api-memory-limit: 500Mi
     stateful-service-replicas: 2
     service-name-1: daemon
-    service-image-1: ethereumoptimism/l2geth:0.5.31
-    service-cpu-limit-1: "4"
-    service-cpu-request-1: "2"
+    service-image-1: #OP_GETH IMAGE
```
Comment (kaladinlight) [inline .circleci/config.yml:537]: Will be released on upgrade day

File: .circleci/config.yml [hunk 3]
```diff
@@ -534,17 +534,17 @@ aliases:
...
+    service-cpu-limit-1: "8"
+    service-cpu-request-1: "4"
     service-memory-limit-1: 24Gi
     service-storage-size-1: 500Gi
-    service-name-2: dtl
-    service-image-2: ethereumoptimism/data-transport-layer:0.5.50
+    service-name-2: op-node
+    service-image-2: #OP_NODE IMAGE
```
Comment (kaladinlight) [inline .circleci/config.yml:543]: Will be released on upgrade day

File: node/coinstacks/optimism/daemon/init.sh [hunk 4]
```diff
@@ -2,73 +2,37 @@
...
-    --mine \
-    --miner.etherbase=$BLOCK_SIGNER_ADDRESS \
-    --miner.gastarget=15000000 \
-    --miner.gaslimit=15000000 \
-    --rpc \
-    --rpcaddr=0.0.0.0 \
-    --rpcapi=eth,rollup,net,web3,debug \
-    --rpcvhosts="*" \
-    --rpccorsdomain="*" \
+    --networkid 420 \
```
Comment (kaladinlight) [inline node/coinstacks/optimism/daemon/init.sh:14]: On upgrade day
```suggestion
    --$NETWORK \
```

File: node/coinstacks/optimism/pulumi/index.ts [hunk 5]
```diff
@@ -14,19 +14,29 @@ export = async (): Promise<Outputs> => {
       case 'daemon':
         return {
           ...service,
+          env: {
+            NETWORK: config.network,
+            SNAPSHOT: 'https://storage.googleapis.com/oplabs-goerli-data/goerli-bedrock.tar', // TODO FINAL SNAPSHOT ENDPOINT
```
Comment (lmyslinski) [inline node/coinstacks/optimism/pulumi/index.ts:19]: I suppose this is also to be updated today

Comment (kaladinlight) [reply node/coinstacks/optimism/pulumi/index.ts:19]: yep, all snapshot, image, and chain refs

**Pull request merged**